builtins/en/facets/partials/output-contracts/base-review-rejection-gate-only-when.md

```
@@ -0,0 +1,2 @@
+- REJECT only when at least one finding exists in `new` with a valid `Authorization Basis`, `persists` under its adjudication-bound definition, or `reopened` with a valid basis (a-d)
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)